src/momento/typing.py

```diff
@@ -13,8 +13,9 @@
 
 # List Types
 TListName = TCollectionName
-TListValuesBytes = List[bytes]
-TListValuesStr = List[str]
+TListValue = Union[str, bytes]
+TListValuesBytes = Sequence[bytes]
+TListValuesStr = Sequence[str]
```

Comment:
Per the discussion, we should accept the broadest types when storing data, but when retrieving the list, give back a concrete list